src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -24,13 +27,13 @@ let currentHeapLock: MonoHeapLock | null = null;
 // The implementations are exactly equivalent to what the global getValue(addr, type) function does,
 // except without having to parse the 'type' parameter, and with less risk of mistakes at the call site
 function getValueI16(ptr: number) {
-  return Module.HEAP16[ptr >> 1];
+  return MONO.getI16(ptr);
```

Comment:
The wrappers here, `getValueI16`, `getValueI32`, etc., no longer seem to do anything - would it be viable to remove them and inline them into their usage sites? We need to keep `readInt16Field` etc, but it could directly call `MONO.getI16`.